_config.yml

```diff
@@ -11,8 +11,19 @@ repository: pactus-project/PIPs
 
 plugins:
   - jekyll-sitemap
+  - jekyll-spaceship
```

Comment:
Do we need to add another plugin, can we just  add `mermaid` plugin??

Reply:
We need just mermaid